spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -401,6 +401,20 @@ public void validate(InjectionPoint injectionPoint);
 
 ==== Firing an event
 
+The method `BeanManager.getEvent()` returns an an instance of `Event<Object>` and without qualifier.
```

Comment:
'@Inject Event<Object>' doesn't have `@Default` qualifier since firing it it won't trigger `@Observes @Default Object ` unless you explicit `select()` the `@Default` qualifier before `fire()`.

So "specified qualifier `@Any`" is better and "with no qualifier" is nearer to the truth.